keps/2941-DRA/examples/problematic-resourceflavors/problematic-rf-cq.yaml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Is this file referenced from the body of the text? If not, then drop it, or make sure referenced.

Reply:
+1, I vote to keep the examples in place that reflect the current agreed upon API, and to link to them (something like [more examples here](./examples/) at the bottom of the `### Workloads` section should suffice.

Reply:
updated.